go.mod

```diff
@@ -28,8 +29,9 @@ require (
 	github.com/muesli/termenv v0.9.0
 	github.com/muhammadmuzzammil1998/jsonc v0.0.0-20201229145248-615b0916ca38
 	github.com/opentracing/opentracing-go v1.1.0
+	github.com/samcoe/go-graphql-client v0.0.0-20211113042829-5271a8a7439c
```

Comment:
This is using my fork but we have already discussed plans to use the `cli` one, please ignore.